crates/sui-indexer/README.md

```diff
@@ -22,7 +22,12 @@ diesel setup
 # and then run 
 diesel migration run
 ```
-2. cargo run under `/sui-indexer`
+2. checkout [the latest devnet release](https://github.com/MystenLabs/sui/releases), otherwise API version mismatch could cause errors
+```sh
+# an example LATEST_RELEASE is: releases/sui-v0.19.0-release
+git checkout <LATEST_RELEASE>
+```
+3. cargo run under `/sui-indexer`
```

Comment:
this reads as though one should type the command:
`cargo run under /sui-indexer`
is it that one should use `cargo run` from the /sui-indexer folder?

Might be easier to format the text as text and then use code/shell for the command only